mod/item.php

```diff
@@ -337,7 +337,7 @@ function item_post(App $a) {
 	$categories = FileTag::listToFile(trim(defaults($_REQUEST, 'category', '')), 'category');
 	$categories_new = $categories;
 
-	if (!empty($filedas)) {
+	if (!empty($filedas) && is_array($filedas)) {
```

Comment:
`$filedas` is only returned by `FileTag::fileToArray` that only returns array, did you get a specific type error? I'd rather you initialized `$filedas = []` before line 330.